css-brochure-form/styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+  font-family: "Brandon Grotesque", sans-serif;
+}
+
+h3 {
+  text-align: center;
+  color: #ec5d57;
+  font-size: 1.1rem;
+  letter-spacing: 0.1rem;
+  margin-bottom: 10px;
+  font-weight: light;
+}
+
+label {
+  padding-left: 0.7em;
+  font-size: 70%;
```

Comment:
I would stick to `px` or `rem` for font sizes.

Reply:
![css-brochure-form rem](https://user-images.githubusercontent.com/112663686/195643320-ef736d74-dadd-4b74-ac4b-726906c214e2.jpg)
Changed font size to rem.